### ISSUE TITLE
except: == "the diaper pattern"

### ISSUE THREAD

**Issue by ThomasWaldmann** [andreasdoerr/impf_termin_tester#9]
in the loop where you iterate over the notifiers, there is an `except: return False`.

i see your intention there, to not ever crash just because some notifier has an issue, but to keep running.

but, there is a problem: the `except:` silently catches all the shit (thus "diaper pattern") and you'll never see it, you'll never know it, you can never fix it. :-)

so the least thing one wants to do there, is some `logging.exception("...")` in the `except` block, so exceptions don't become invisible.

if we then see some "usual" exceptions that we specifically want to deal with, we can add more specific exception handler (likely inside the notifier code), so the screen output does not get ugly for frequently seen issues (like temp. internet connection issues or so).

**Comment by ThomasWaldmann** [comment]
related: #10

**Comment by andreasdoerr** [comment]
Agreed. Makes sense. There is a general exception handling preventing the whole runner to crash. The notifier exception handling is indeed a bit aggressive;)
